freqtrade/exchange/exchange.py

```diff
@@ -1704,6 +1746,141 @@ def set_margin_mode(self, pair: str, collateral: Collateral, params: dict = {}):
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
 
+    @retrier
+    def _get_mark_price_history(
+        self,
+        pair: str,
+        since: int
+    ) -> Dict:
+        """
+            Get's the mark price history for a pair
```

Comment:
I'm honestly not sure why you're indenting docstrings. 
i've never seen it indented before - and i really don't like how it looks (it also makes your lines shorter ... even though that's not relevant in this case).

also the [Python PEP](https://www.python.org/dev/peps/pep-0257/) about docstring convention doesn't indent the docstring further ... so i'd like them to be aligned with the first quote.

Reply:
> I'm honestly not sure why you're indenting docstrings.

It makes them collapseable

Reply:
That's fine for your environment - for commits, please dedent them.

I recognize now that i made a mistake in merging PR's that had docstrings this way, so i'll fix it for develop and feat/short.